powershell/cmdlets/class.ts

```diff
@@ -2350,4 +2299,76 @@ export class CmdletClass extends Class {
       this.add(new Attribute(NotSuggestDefaultParameterSetAttribute));
     }
   }
+
+  private inferBreakingChangeAttribute(breakingChange: any, outputTypes: Set<string>) : Attribute | null {
+    if (breakingChange.cmdlet) {
+      const parameters = [];
+      if (!breakingChange.cmdlet.deprecateByVersion || !breakingChange.cmdlet.deprecateByAzVersion) {
+        throw new Error('Cmdlet breaking change requires both \'deprecateByVersion\' and \'deprecateByAzVersion\', please refer to https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/breakingchange-for-autogen-module.md for more details.');
+      }
+
+      if (!breakingChange.cmdlet.replacement && breakingChange.cmdlet.changeDescription) {
```

Comment:
The core logic of this PR is this if-else. In the case when user preannounces a breaking change with `changeDescription`, we should prioritize showing it instead of the hardcoded "The cmdlet is being deprecated without replacement".